### PR TITLE
BE-41 Enable recently introduced Lint/DuplicateMatchPattern cop

### PR DESCRIPTION
**Pull request by chiibis**
### References
[help]: # (Add link/s to every reference related to this pull request: issue, migrations, another PRs, ...)
* **Issue:** https://hellogetsafe.atlassian.net/browse/BE-41

### What is the goal? How is it being implemented?
[help]: # (Provide a description of the overall goal and a description of the implementation.)
Small update to prevent "new cops" warning

### CODE REVIEW

File: default.yml [hunk 1]
```diff
@@ -241,10 +243,8 @@ Lint/ToEnumArguments:
 Lint/UselessRuby2Keywords:
   Enabled: false
 Lint/DuplicateMagicComment:
-  Enabled: false
-Lint/UselessRescue:
-  Enabled: false
-Lint/DuplicateMagicComment:
```
Comment (chiibis) [inline default.yml:247]: Duplicated rules in config

File: default.yml [hunk 2]
```diff
@@ -241,10 +243,8 @@ Lint/ToEnumArguments:
 Lint/UselessRuby2Keywords:
   Enabled: false
 Lint/DuplicateMagicComment:
-  Enabled: false
-Lint/UselessRescue:
```
Comment (chiibis) [inline default.yml:245]: We accidentally declared these rules twice, I'm removing second copy.

File: nxt_cop.gemspec [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
```
Comment (shalvah-gs) [inline nxt_cop.gemspec:1]: FYI: we can remove this magic comment 👀 . Ruby's default encoding has been UTF-8 for a long time.

**Review by shalvah-gs — APPROVED** [review]

File: nxt_cop.gemspec (hunk 3 above)
Comment (chiibis) [reply nxt_cop.gemspec]: Good idea, thanks! 

Removed

**Pull request merged**